src/constants/Typography.ts

```diff
@@ -0,0 +1,44 @@
+export type Sizes = 'xs' | 'sm' | 'md' | 'lg' | 'xl' | 'xxl' | 'super' | 'max';
+
+export const fontSizes: Record<Sizes, string> = {
+    xs: '10px',
+    sm: '12px',
+    md: '14px',
+    lg: '16px',
```

Comment:
I would remove `px` for all the values. RN by default doesn't use `px`, just number is enough